ui/page/components/seed_restore_page.go

```diff
@@ -448,7 +448,7 @@ func (pg *SeedRestore) verifySeeds() bool {
 
 	if isValid {
 		pg.seedPhrase = seedphrase
-		if !sharedW.VerifySeed(pg.seedPhrase, pg.walletType, pg.getWordSeedType()) {
+		if !sharedW.VerifyMnemonic(pg.seedPhrase, pg.walletType, pg.getWordSeedType()) {
```

Comment:
It looks like there is an extra space in the pg.seedPhrase that is preventing the checksum from verifying. Need to trim that before doing anything with it. This is causing the verification to fail for me.